patterns/1-initial/creating_an_innersource_strategy.md

```diff
@@ -0,0 +1,78 @@
+
+## Forces
+
+TODO
+
+<!--
+What makes the problem difficult?
+What are the trade-offs?
+These are constraints that **can be changed** at a cost.
+The solution might change one or more of these forces in order to solve the problem, while also in-turn changing the context.
+-->
+
+## Solutions
+
+Create an InnerSource strategy. Such a document describes how the application of InnerSource relates to the goals and business strategy of your organization. It explains why InnerSource is important (not only in an abstract way but specifically to your organization), why you should leverage it, what goals your organization should have with respect to InnerSource, and what actions to perform to achieve those.
+
+A high level table of contents of an InnerSource strategy document could look like so:
+
+- Executive summary
+- What InnerSource is
+- Why InnerSource is important for our organization
+- Current status of InnerSource in your organization
+- Target state
+- Recommended actions (to reach the target state)
+
+Such a strategy is effective in persuading colleagues who might not have seen the value of InnerSource so far. Also discussions with management will be easier if you can draw clear connections between your goals and recommended activities to establish InnerSource and the goals and business strategy of your organization. It can be helpful to mention what other organizations (your peer organizations and/or competitors, for example) do with respect to InnerSource. The sections "target state" and "recommended actions" can be used as input and guardrails for the work of your InnerSource program.
+
+If your organization has an approval process for strategy document, it is recommended to get an approval for your InnerSource strategy. Such a process can trigger discussions about InnerSource in your organization, raise awareness and make such a strategy document more "official".
+
+## Resulting Context
+
+TODO
+
+<!--
+What is the situation after the problem has been solved?
+The original context is changed indirectly by way of the solution.
+Often this section can include discussion of the next possible Patterns/problems introduced.
+This section can be short in content - the solution may not introduce new problems or change much context.
+-->
+
+## Known Instances (optional)
+
+SAP has an InnerSource strategy (see blog post "[Cultivating InnerSource at SAP](https://community.sap.com/t5/open-source-blogs/cultivating-innersource-at-sap/ba-p/13575274)")
+
+TODO
+
+<!--
+Where has this been seen before?
+Helps to reinforce that this is a REAL pattern and that you match the context.
+
+May mention:
+
+* A particular business
+* Anonymized instances ex: "3 companies have proven that this is a good solution" or "A large financial services org...".
+-->
```

Comment:
```suggestion
```

For status "initial" a known instance isn't even required. So even having the SAP example in here is already amazing. Therefore I recommend to remove the TODO+comments here.